en/identity-server/7.0.0/docs/deploy/security/asymmetric-encryption/index.md

```diff
@@ -1 +1 @@
-{% include "../../../../../../includes/guides/encryption/asymmetric-encryption.md" %}
+{% include "../../../../../../includes/deploy/security/asymmetric-encryption.md" %}
```

Comment:
includes/deploy/security/asymmetric-encryption/index.md

Reply:
fixed with: https://github.com/wso2/docs-is/pull/4839